### DISCUSSION TITLE
Seems to be a bug that Keynote audio is for some videos not transferred (muted)

### DISCUSSION THREAD

**Discussion by slajerek** [ExistentialAudio/BlackHole#820] [General]
I have a Keynote presentation with videos, that some of them work correctly and audio is transferred to OBS, and some not. I will post here a minimal working scenario and a Keynote file. I wonder however how it is possible, would it be possible that Keynote changes audio out parameters depending on the video codec that are not supported by Blackhole?

**Comment by devinroth** [comment]
Unlikely related to video codec. Probably has more to do with Apple blocking virtual audio devices. Take a look at the FaceTime issue. I bet it’s the same thing.

Devin

**Comment by slajerek** [comment]
Interesting, you mean that Apple tries to prove that I do not own a simple jack cable that can be mixed in my 16 channels analog mixer?

**Comment by devinroth** [comment]
Yup. Ever try to take a screenshot of Disney+ or AppleTV? Although oddly in that situation we can still capture the audio.

If you’re up for it, try building a custom version of BlackHole, completely change the name to something else and make the device type as USB instead of virtual. If that fixes the issue we know exactly what Apple is doing here.

Devin
